homeworks/vladislav.buzunar_buzyaka/5-arrays/arrays.js

```diff
@@ -0,0 +1,39 @@
+let branchs;
+let needlesOnTree;
+
+function christmasTree(height) {
+    branchs = '';
```

Comment:
Move the initialization of the variable into the function, please. The same for the second variable.

Reply:
Done!